platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/preview/pdf/StoredFilePdfPreviewPopup.java

```diff
@@ -25,19 +25,19 @@ public StoredFilePdfPreviewPopup(AGStoredFile file) {
 	public StoredFilePdfPreviewPopup(AGStoredFile file, ICssLength imageInitialMaxWidth) {
 
 		super(file);
-		List<DomImageViewerImage> previewImages = getPreviewImages(file, imageInitialMaxWidth);
+		List<IResource> previewImages = getPreviewImages(file);
 		if (!previewImages.isEmpty()) {
-			appendChild(new DomImageViewer(previewImages));
+			appendChild(new DomImageViewer(previewImages, imageInitialMaxWidth));
 		} else {
 			appendChild(new DomMessageDiv(DomMessageType.WARNING, EmfI18n.NO_IMAGE_FOUND_FOR_ARG1.toDisplay(file.getFileName())));
 		}
 	}
 
-	private List<DomImageViewerImage> getPreviewImages(AGStoredFile file, ICssLength maxWidth) {
+	private List<IResource> getPreviewImages(AGStoredFile file) {
 
 		return Optional//
 			.of(file)
-			.map(it -> StoredFilePdfToZoomableImagesConverter.convertPagesToImages(file, maxWidth))
+			.map(it -> StoredFilePdfRenderer.renderPages(file))
```

Comment:
Odd lambda definition. `it` and `file` are the same. We should probably just use a method reference.

Reply:
not my code, I removed the complete method, cause it's useless